app/controllers/admin/departments_controller.rb

```diff
@@ -17,7 +17,7 @@ class DepartmentsController < Admin::ApplicationController
     # for more information
 
     def find_resource(param)
-      Department.find_by! slug: param
+      Department.find param
```

Comment:
ActiveRecord idiosyncracy: #find throws an exception when it can't find a record. There is no #find! method.